tests/ts_api.erl

```diff
@@ -29,74 +31,273 @@
 %---------------------------------------------------------------------
 
 confirm() ->
-    DDL  = ts_api_util:get_ddl(api),
-    Data = ts_api_util:get_data(api),
-    C    = ts_api_util:setup_cluster(single, normal, DDL, Data),
+    DDL  = ts_util:get_ddl(api),
+    Data = ts_util:get_data(api),
+    ClusterConn = {_Cluster, Conn} = ts_util:cluster_and_connect(single),
+    ?assertEqual(ok, ts_util:ts_put(ClusterConn, normal, DDL, Data)),
 
-    confirm_GtOps(C),
-    confirm_GtEqOps(C),
-    confirm_LtOps(C),
-    confirm_LtEqOps(C),
-    confirm_EqOps(C),
-    confirm_NeqOps(C).
+    confirm_GtOps(Conn),
+    confirm_GtEqOps(Conn),
+    confirm_LtOps(Conn),
+    confirm_LtEqOps(Conn),
+    confirm_EqOps(Conn),
+    confirm_NeqOps(Conn),
+    pass.
 
 %------------------------------------------------------------
 % Test > Ops
 %------------------------------------------------------------
 
 confirm_GtOps(C) ->
```

Comment:
Function names are all over the place here, probably a relic of Andy's previous Erlang lives where he worked in evil environments where they did weird stuff. Would be nice to see a little less erraticism but I could live with it for this PR.